styleguide/components/31-molecules/crisis-communication-block/_crisis-communication-block.scss

```diff
@@ -0,0 +1,14 @@
+.crisis-communication-block {
+  p {
+    margin: 0;
+    font-weight: 700;
+  }
+
+  .icon-list {
+    margin-top: .8rem;
+  }
+
+  ul:last-of-type {
+    margin-bottom: 0;
```

Comment:
Maybe we should move this to box level: *:last-of-type
@meirege ?

Reply:
This change breaks the whitespace between some components. Also this breaks the margin of the title...
A title has it's margin, I wouldn't change that here, we don't do it elsewhere so...